edge/conf/edge.yaml

```diff
@@ -44,3 +45,9 @@ edged:
     maximum-dead-containers-per-container: 1
     docker-address: unix:///var/run/docker.sock
     version: 2.0.0
+    runtime-type: docker
```

Comment:
should this be "remote" ?
or if we are specifying remote endpoint configs below , then runtime type is optional ?

Reply:
Can be docker/remote...If nothing is specified will be docker by default